### PR TITLE
Alinux e4s pipeline

### PR DESCRIPTION
**Pull request by bollig**
Reattempt to revert the revert of #29522

### CODE REVIEW

File: share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml [hunk 1]
```diff
@@ -52,12 +52,22 @@ default:
       - runner_system_failure
       - stuck_or_timeout_failure
 
+.generate-aarch64:
+  extends: [ ".generate" ]
+  tags: ["spack", "public", "medium", "aarch64"]
```
Comment (scottwittenburg) [inline share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml]: Whoops, I forgot an `aws` in here.  Our new policy is to run all pipeline generation jobs on runners under our control.  Fix coming shortly.

**Review by scottwittenburg — DISMISSED** [review]
LGTM pending CI

**Comment by bollig** [comment]
@spackbot hello

**Comment by bollig** [comment]
@spackbot help

**Comment by bollig** [comment]
@spackbot re-run pipeline

**Comment by scottwittenburg** [comment]
@spackbot run pipeline

**Comment by alalazo** [comment]
Are we still interested in this?

**Comment by bollig** [comment]
Closed in favor of modern lines that @stephenmsachs has pushed.

**Pull request closed without merging**